doc/user/inputs/settings.rst

```diff
@@ -132,3 +132,97 @@ These settings come with a few caveats:
+
+    2. 50% power for 30 days, split into two segments of 15 days each, followed by 70 days shutdown (i.e. 15% capacity)
+
+    3. 100% power for 93 days, split into two segments of 46.5 days each, followed by 7 days shutdown (i.e. 93% capacity)
+
+In each cycle, criticality calculations will be performed at 3 nodes evenly-spaced through the uptime portion of the cycle (i.e. ``availabilityFactor``*``powerFraction``), without option for changing node spacing or frequency.
+This input format can be useful for quick scoping and certain types of real analyses, but clearly has its limitations.
+
+To overcome these limitations, the detailed cycle history, consisting of the ``cycles`` setting may be specified instead.
+For each cycle, an entry to the ``cycles`` list is made with the following optional fields: 
+    
+    * ``name``
+    * ``power fractions``
+    * ``cumulative days``, ``step days``, or ``burn steps`` + ``cycle length``
+    * ``availability factor``
+
+An example detailed cycle history employing all of these fields could look like::
+
+    power: 1000000
+    nCycles: 4
+    cycles: 
+      - name: A
+        step days: [1, 1, 98]
+        power fractions: [0.1, 0.2, 1]
+        availability factor: 0.1
+      - name: B
+        cumulative days: [2, 72, 78, 86]
+        power fractions: [0.2, 1.0, 0.95, 0.93]
+      - name: C
+        step days: [5, R5]
+        power fractions: [1, R5]
+      - cycle length: 100
+        burn steps: 2
+        availability factor: 0.9
+
+Note that repeated values in a list may be again be entered using the shorthand notation for ``step days``, ``power fractions``, and ``availability factors`` (though not ``cumulative days`` because entries must be monotonically increasing).
+
+Such a scheme would define the following cycles:
+
+    1. A 2 day power ramp followed by full power operations for 98 days, with three nodes clustered during the ramp and another at the end of the cycle, followed by 900 days of shutdown
+
+    2. A 2 day power ramp followed by a prolonged period at full power and then a slight power reduction for the last 14 days in the cycle
+
+    3. Constant full-power operation for 30 days split into six even increments
+
+    4. Constant full-power operation for 90 days, split into two equal-length 45 day segments, followed by 10 days of downtime
+
+As can be seen, the detailed cycle history option provides much greated flexibility for simulating realistic operations, particularly power ramps or scenarios that call for unevenly spaced burnup nodes, such as xenon buildup in the early period of thermal reactor operations.
+
+.. note:: Although the detailed cycle history option allows for powers to change within each cycle, it should be noted that the power over each step is still considered to be constant.
+
+.. note:: The detailed cycle history may not be used for equilibrium calculations at this time.
```

Comment:
Can you show me where this breaks down?

Thanks!

Reply:
It doesn't actually break down explicitly in the code. But the issue is more a theoretical one. The equilibrium cycle has only one cycle, so using the detailed `cycles` input where each cycle can have a unique history doesn't make sense. So instead of checking their `cycles` input to make sure that it has only 1 cycle defined, we just say they can't use the `cycles` input in equilibrium cases. 

I had a short discussion with @ntouran about this, and this seemed like the best, least confusing path.

Reply:
That makes sense, thanks for taking the time to explain it.

Hmm... You know, in `SettingsValidation` we can check the `RunType`. So, we could explicitly check if the `cycles` are valid in the uncommon case where `cs["runType"] == operators.RunTypes.EQUILIBRIUM`.

But just because we _can_ do a thing, doesn't mean we _should_ do a thing.

I can't decide if that's a good idea or not.  Do you have an opinion?

Reply:
To be honest, I _do_ think that it would be good to allow for the detailed `cycles` input to be used for equilibrium cases, in particular because it would probably be necessary for thermal-spectrum reactors to converge to an accurate equilibrium cycle.

I didn't allow for it, though, because my initial survey made it appear kinda difficult to make it work in our downstream projects. I can reevaluate that, though.

If we _did_ allow the use of the detailed `cycles` with equilibrium runs, I guess the only requirement that would need to be checked is that the `cycles` entry has only 1 cycle's information in it.

So I'll look into this a little more tomorrow. And if it looks like it is going to appreciably stall getting this change into the framework, then I'll just punt it into a new ticket for future work.

Reply:
>  I guess the only requirement that would need to be checked is that the cycles entry has only 1 cycle's information in it.

Yeah, man. That's all I meant.

But it's at your discretion either way.